pkgs/development/python-modules/vulcan-api/default.nix

```
@@ -17,7 +17,7 @@
 
 buildPythonPackage rec {
   pname = "vulcan-api";
-  version = "2.3.2";
+  version = "2.4.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `build-system`